src/main/java/doob/model/powerup/PlayerSpeedPowerUp.java

```diff
@@ -25,7 +24,7 @@ public void onActivate(Level level, Player player) {
     }
 
     /**
-     * Does nothing.
+     * Sets player speed back to initial value.
      * @param level the level the power-up is in.
```

Comment:
Not all of them are the same. Some Power-ups have nothing to do on deactivate. But for consistency, I will change all comments on Power-ups that do change onDeactivate to the same syntax.